import "MSG_DS_RA_IMPORT_TASK_SEND.proto";
message MSG_DS_RA_DELTA_GET
{
    required uint32 taskID = 1;
    repeated IMP_DB_INFO dbInfo = 2;
    optional uint32 rowKeyMax = 3;
}

message MSG_RA_DS_DELTA_GET_ACK
{
    required uint32 taskID = 1;
    required uint32 dbID = 2;
    required string tableName = 3;
    optional uint32 rowKeyMax = 4;
    repeated COL_DATA colValue = 5;
}
